delivery_service.rb

```diff
@@ -0,0 +1,40 @@
+class DeliveryService
+  attr_accessor :cars_count, :bikes_count, :auto_park
+
+  require_relative 'car'
+  require_relative 'bike'
+  require_relative 'constants'
+
+  def initialize(cars_count = 6, bikes_count = 8)
```

Comment:
Magic numbers here, feel free to add them as a constants. If you want cars_count to default to 6 you can put DEFAULT_CARS_COUNT = 6 in constants and simply use:
```
@cars_count ||= Constants::DEFAULT_CARS_COUNT
```
that will allow you to set default cars count if none provided on `DeliveryService.new` instance initialization.

Reply:
Constants are used in the definition parameters line. Is it ok? Otherwise I should add parameters when use this class